src/main/java/core/basesyntax/fruittransact/PurchaseService.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.fruittransact;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+
+public class PurchaseService implements FruitService {
+    private FruitDao fruitDao = new FruitDaoImpl();
+
+    @Override
+    public void handle(String name, int amount) {
+        fruitDao.remove(name, amount);

```

Comment:
If you have enough fruits, you can buy them
otherwise, you should throw the "Not enough fruitName to purchase" exception